### ISSUE TITLE
WebPages Razor Scaffolding

### ISSUE THREAD

**Issue by troncomputers** [aspnet/Mvc#6640]
Hi!
I have a problem with scaffolding ASP .NET Core Razor WebPages models.
Configured from this link:
[Adding a model to a Razor Pages app](https://docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/model)

When I use this command:
```
dotnet aspnet-codegenerator -?
```
Under the "Code Generators" section, I have only this:
```
Code Generators:
view
controller
area
```
![code_generators](https://user-images.githubusercontent.com/29543667/29127300-fafd4216-7d20-11e7-85f7-3d32e3e49775.png)

Why I don't have 'razorpage' code generator?

.csproj file content
```
﻿<Project Sdk="Microsoft.NET.Sdk.Web">

  <PropertyGroup>
    <TargetFramework>netcoreapp2.0</TargetFramework>
    <MvcRazorCompileOnPublish>true</MvcRazorCompileOnPublish>
  </PropertyGroup>

  <ItemGroup>
    <Content Remove="wwwroot/raport_3e04f27c-9768-4adb-93b2-3e54f8023e10.pdf" />
  </ItemGroup>

  <ItemGroup>
    <Content Include="wwwroot\css\style.css" />
    <Content Include="wwwroot\images\logo_optimo.png" />
    <Content Include="wwwroot\images\logo_optimo_transparent.png" />
  </ItemGroup>

  <ItemGroup>
    <PackageReference Include="CsvHelper" Version="2.16.3" />
    <PackageReference Include="EPPlus.Core" Version="1.5.1" />
    <PackageReference Include="Microsoft.AspNet.Razor" Version="3.2.3" />
    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.0-preview1-final" />
    <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="2.0.0-preview1-final" />
  </ItemGroup>
  <ItemGroup>
    <DotNetCliToolReference Include="Microsoft.Extensions.SecretManager.Tools" Version="2.0.0-preview1-final" />
    <DotNetCliToolReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Tools" Version="2.0.0-preview1-final" />
  </ItemGroup>
  <ItemGroup>
    <Folder Include="Pages\Account\" />
    <Folder Include="Pages\Admin\Account\" />
    <Folder Include="Pages\Raporty2\" />
    <Folder Include="Pages\Raporty\" />
    <Folder Include="Pages\Zdarzenie\" />
    <Folder Include="Pages\Wydruki\" />
    <Folder Include="Pages\Przeglad\" />
    <Folder Include="Pages\Opcje\" />
    <Folder Include="Pages\Urzadzenie\" />
    <Folder Include="Pages\Podatnik\" />
  </ItemGroup>
  <ItemGroup>
    <Compile Update="Properties\Resources.Designer.cs">
      <DesignTime>True</DesignTime>
      <AutoGen>True</AutoGen>
      <DependentUpon>Resources.resx</DependentUpon>
    </Compile>
  </ItemGroup>
  <ItemGroup>
    <EmbeddedResource Update="Properties\Resources.resx">
      <Generator>ResXFileCodeGenerator</Generator>
      <LastGenOutput>Resources.Designer.cs</LastGenOutput>
    </EmbeddedResource>
  </ItemGroup>

</Project>
```

**Comment by pranavkm** [comment]
@prafullbhosale?

**Comment by prafullbhosale** [comment]
Razor page scaffolding is not publicly available yet.
It will be included in the 2.0.0 versions of scaffolding packages. 

@troncomputers, you would need to update `Microsoft.VisualStudio.Web.CodeGeneration.Design` to 2.0.0 once it is available on NuGet.org

**Comment by pranavkm** [comment]
Close since the RTM bits should address this issue.
